### PR TITLE
Fix(web-react, web-twig): PasswordToggle disabled #DS-1478

### PR DESCRIPTION
**Pull request by pavelklibani**
### Issue reference

[Password Toggle button should be disabled if the TextField is disabled](https://jira.almacareer.tech/browse/DS-1478)

### CODE REVIEW

File: packages/web-twig/src/Resources/components/TextField/__tests__/__fixtures__/textFieldPasswordToggle.twig [hunk 1]
```diff
@@ -4,3 +4,11 @@
...
     name="passwordToggle"
 />
+
+<TextField
+    hasPasswordToggle
+    id="password-toggle"
+    label="Text field with password toggle"
+    name="passwordToggle"
+    isDisabled
+/>
```
Comment (crishpeen) [inline packages/web-twig/src/Resources/components/TextField/__tests__/__fixtures__/textFieldPasswordToggle.twig:14]: please add empty line to the end of the file

File: packages/web-react/src/components/TextField/__tests__/TextField.test.tsx [hunk 2]
```diff
@@ -22,96 +22,94 @@ describe('TextField', () => {
...
+  });
+
+  describe('hasPasswordToggle', () => {
     it('should have password toggle', () => {
-      const dom = render(<TextField id="textfield" label="Label" hasPasswordToggle />);
+      render(<TextField id="textfield" label="Label" hasPasswordToggle />);
 
-      const element = dom.container.querySelector('.TextField__passwordToggle') as HTMLElement;
-      expect(element).toBeInTheDocument();
+      expect(document.querySelector('.TextField__passwordToggle')).toBeInTheDocument();
```
Comment (crishpeen) [inline packages/web-react/src/components/TextField/__tests__/TextField.test.tsx:66]: queryselector?

**Review by crishpeen — APPROVED** [review]

**Review by literat — APPROVED** [review]
LGTM 👍

File: packages/web-twig/src/Resources/components/TextField/__tests__/__fixtures__/textFieldPasswordToggle.twig (hunk 1 above)
Comment (pavelklibani) [reply packages/web-twig/src/Resources/components/TextField/__tests__/__fixtures__/textFieldPasswordToggle.twig]: new line added

File: packages/web-react/src/components/TextField/__tests__/TextField.test.tsx (hunk 2 above)
Comment (pavelklibani) [reply packages/web-react/src/components/TextField/__tests__/TextField.test.tsx]: changed to `parentElement`

**Review by crishpeen — APPROVED** [review]

**Review by curdaj — APPROVED** [review]

**Review by literat — CHANGES_REQUESTED** [review]
Disabled TextField can be edited and focused.

**Comment by pavelklibani** [comment]
> Disabled TextField can be edited and focused.

fixed

**Review by literat — APPROVED** [review]
👍

**Pull request merged**